PA_Feature_Detecting.md

```diff
@@ -101,8 +101,8 @@ const reportingTimeout = auctionConfig.reportingTimeout ?
 
 From the context of a web page, whether custom reporting timeout is enabled can be queried as follows:
 ```
-const reportingTimeoutEnabled = navigator.protectedAudience ?
-    navigator.protectedAudience.queryFeatureSupport("reportingTimeout") : false;
+const reportingTimeoutEnabled = navigator.protectedAudience &&
```

Comment:
If you want to get fancy you can use `navigator.protectedAudience?.queryFeatureSupport("reportingTimeout")`

Reply:
Thanks for something fancy! I'd like to keep it consistent with other examples in the doc, and we can make them fancier in the future.